metron-platform/metron-api/src/main/resources/META-INF/NOTICE

```diff
@@ -0,0 +1,87 @@
+
+metron-api
+Copyright 2006-2016 The Apache Software Foundation
+
+This product includes software developed at
+The Apache Software Foundation (http://www.apache.org/).
+
+This product includes software developed for Orekit by
+CS Systèmes d'Information (http://www.c-s.fr/)
+Copyright 2010-2012 CS Systèmes d'Information
+
+                            The Netty Project
+                            =================
+
+Please visit the Netty web site for more information:
+
+  * http://netty.io/
+
+Copyright 2011 The Netty Project
+
+This product includes software developed by
+Joda.org (http://www.joda.org/).
+
+Spring Framework 3.2.6.RELEASE
+Copyright (c) 2002-2013 Pivotal, Inc.
+
+storm-kafka
+Copyright 2016 The Apache Software Foundation
+
+Curator Framework
+Copyright 2011-2016 The Apache Software Foundation
+
+Google Guice - Core Library
+Copyright 2006-2011 Google, Inc.
+
+Google Guice - Extensions - Servlet
+Copyright 2006-2011 Google, Inc.
+
+HBase - Common
+Copyright 2015 The Apache Software Foundation
+
+HBase - Protocol
+Copyright 2015 The Apache Software Foundation
+
+HBase - Annotations
+Copyright 2015 The Apache Software Foundation
+
+Commons Logging
+Copyright 2001-2007 The Apache Software Foundation
+
+Apache Avro
+Copyright 2009-2013 The Apache Software Foundation
+
+ApacheDS Protocol Kerberos Codec
+Copyright 2003-2013 The Apache Software Foundation
+
+ApacheDS I18n
+Copyright 2003-2013 The Apache Software Foundation
+
+Apache Directory API ASN.1 API
+Copyright 2003-2013 The Apache Software Foundation
+
+Apache Directory LDAP API Utilities
+Copyright 2003-2013 The Apache Software Foundation
+
+Curator Client
+Copyright 2011-2015 The Apache Software Foundation
+
+Curator Recipes
+Copyright 2011-2015 The Apache Software Foundation
+
+htrace-core
+Copyright 2015 The Apache Software Foundation
+
+   Apache Xerces Java
+   Copyright 1999-2007 The Apache Software Foundation
+
+   Portions of this software were originally based on the following:
```

Comment:
Just to be clear, I would retain this in notice (IBM, Sun, Paul Eng via iClick, and ASF), but the other ASF mentions below the Netty line can be removed.